modules/custom/az_news/config/install/views.view.az_news.yml

```diff
@@ -549,6 +578,653 @@ display:
         - 'user.node_grants:view'
         - user.permissions
       tags: {  }
+  block_1:
```

Comment:
Should add a more descriptive display id like this one: https://github.com/az-digital/az_quickstart/pull/1410/files#diff-0859d416184da2cd73ce4dadd648c939bbb1bdc62c00aa627a026265c124314fL488